app/code/Magento/Developer/Test/Unit/Console/Command/GeneratePatchCommandTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Test\Unit\Console\Command;
+
+use Magento\Developer\Console\Command\GeneratePatchCommand;
+use Magento\Framework\Component\ComponentRegistrar;
+use Magento\Framework\Filesystem\Directory\Read;
+use Magento\Framework\Filesystem\Directory\ReadFactory;
+use Magento\Framework\Filesystem\Directory\Write;
+use Magento\Framework\Filesystem\Directory\WriteFactory;
+use Magento\Framework\Filesystem\DirectoryList;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class GeneratePatchCommandTest extends TestCase
+{
+    /**
+     * @var ComponentRegistrar|MockObject
+     */
+    private $componentRegistrarMock;
+
+    /**
+     * @var DirectoryList|MockObject
+     */
+    private $directoryListMock;
+
+    /**
+     * @var ReadFactory|MockObject
+     */
+    private $readFactoryMock;
+
+    /**
+     * @var WriteFactory|MockObject
+     */
+    private $writeFactoryMock;
+
+    /**
+     * @var GeneratePatchCommand|MockObject
+     */
+    private $command;
+
+    protected function setUp()
+    {
+        $this->componentRegistrarMock = $this->createMock(ComponentRegistrar::class);
+        $this->directoryListMock = $this->createMock(DirectoryList::class);
+        $this->readFactoryMock = $this->createMock(ReadFactory::class);
+        $this->writeFactoryMock = $this->createMock(WriteFactory::class);
+
+        $this->command = new GeneratePatchCommand(
+            $this->componentRegistrarMock,
+            $this->directoryListMock,
+            $this->readFactoryMock,
+            $this->writeFactoryMock
+        );
+    }
+
+    public function testExecute()
+    {
+        $this->componentRegistrarMock->expects($this->once())
+            ->method('getPath')
+            ->with('module', 'Vendor_Module')
+            ->willReturn('/long/path/to/Vendor/Module');
+
+        $read = $this->createMock(Read::class);
```

Comment:
Hi @korostii. Thank you for fixing the issues mentioned earlier. There was a comment from me regarding suffixing mock properties with Mock suffix. The same applies to the variables. This suffix improves code readability by letting know straight away what object we are working on mock or a real instance.

Just a recommendation for the future 😉